include/CppUTest/CppUTestConfig.h

```diff
@@ -41,6 +41,7 @@
  */
 
 #ifdef __clang__
+ #define NEEDS_DISABLE_UNREACHABLE_CODE 1
```

Comment:
This one can be deleted again, right? It isn't used now?

Reply:
Right!